multi-node/aws/pkg/cluster/stack_template.go

```diff
@@ -223,6 +224,7 @@ func StackTemplateBody(defaultArtifactURL string) (string, error) {
 				map[string]interface{}{"IpProtocol": sgProtoUDP, "FromPort": 0, "ToPort": sgPortMax, "CidrIp": sgAllIPs},
 			},
 			"SecurityGroupIngress": []map[string]interface{}{
+				map[string]interface{}{"IpProtocol": sgProtoICMP, "FromPort": 3, "ToPort": -1, "CidrIp": sgAllIPs},
```

Comment:
With AWS security groups and `IpProtocol=icmp`, the `FromPort` and `ToPort` parameters take different meanings. `FromPort` is the ICMP type, and `ToPort` is the ICMP code. In both, `-1` means any. So We allow ICMP type 3 (destination unreachable), any code.